language/parser.js

```diff
@@ -126,6 +126,7 @@ export default class Parser {
 	 */
   static getIncludeFromDirective(line) {
     if (line.includes(`*`)) return; // Likely comment
+    if (line.includes(`//`)) return; // Likely comment
```

Comment:
@worksofliam Aren't you returning even if the `//` is _after_ the include - which you handle below..?

Reply:
Believe it or not it wasn't captured in all cases, for example:

```
// /copy a
```